<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QDialog" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWebView" name="webView">
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aSavePDF">
   <property name="icon">
    <iconset resource="../gcu.qrc">
     <normaloff>:/img/application-pdf.png</normaloff>:/img/application-pdf.png</iconset>
   </property>
   <property name="text">
    <string>Save PDF</string>
   </property>
   <property name="toolTip">
    <string>Save PDF</string>
   </property>
  </action>
  <action name="aSaveHTML">
   <property name="icon">
    <iconset resource="../gcu.qrc">
     <normaloff>:/img/text-html.png</normaloff>:/img/text-html.png</iconset>
   </property>
   <property name="text">
    <string>Save HTML</string>
   </property>
   <property name="toolTip">
    <string>Save HTML</string>
   </property>
  </action>
  <action name="aPrint">
   <property name="icon">
    <iconset resource="../gcu.qrc">
     <normaloff>:/img/printer.png</normaloff>:/img/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
  </action>
  <action name="aEditTemplate">
   <property name="icon">
    <iconset resource="../gcu.qrc">
     <normaloff>:/img/configure.png</normaloff>:/img/configure.png</iconset>
   </property>
   <property name="text">
    <string>Edit Template</string>
   </property>
   <property name="toolTip">
    <string>Edit Template</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gcu.qrc"/>
  <include location="../gcu.qrc"/>
  <include location="../gcu.qrc"/>
  <include location="../gcu.qrc"/>
  <include location="../gcu.qrc"/>
 </resources>
 <connections/>
</ui>
